mlflow/utils/requirements_utils.py

```diff
@@ -271,6 +269,39 @@ def _capture_imported_modules(model_uri, flavor):
         # resolution in a subprocess based on PATH entries.
         main_env["PATH"] = "/usr/sbin:/sbin:" + main_env["PATH"]
 
+        if flavor == mlflow.transformers.FLAVOR_NAME:
+            # Lazily import `_capture_transformers_module` here to avoid circular imports.
+            from mlflow.utils import _capture_transformers_modules
+
+            for module_to_throw in ["tensorflow", "torch"]:
+                try:
+                    _run_command(
+                        [
+                            sys.executable,
+                            _capture_transformers_modules.__file__,
```

Comment:
I suggest to use command like `{sys.executable} -m {module_name} {args ...}`

Reply:
why?

Reply:
not a big deal, I think `python -m module` and `python {module.__file__}` should be equivalent. we can keep current code